test/blackbox/test_case_optimal_pmtu_01/node_sim_nut.c

```diff
@@ -45,26 +45,26 @@ static void node_status_cb(meshlink_handle_t *mesh, meshlink_node_t *node,
 static void channel_receive_cb(meshlink_handle_t *mesh, meshlink_channel_t *channel, const void *dat, size_t len);
 static bool channel_accept(meshlink_handle_t *mesh, meshlink_channel_t *channel, uint16_t port, const void *dat, size_t len);
 
-pmtu_attr_t node_pmtu[2];
+pmtu_attr_t *node_pmtu[2];
 static time_t node_shutdown_time = 0;
 static bool mtu_set = true;
 
-static void print_mtu_calc(pmtu_attr_t node_pmtu) {
-	fprintf(stderr, "MTU size : %d\n", node_pmtu.mtu_size);
-	fprintf(stderr, "Probes took for calculating PMTU discovery : %d\n", node_pmtu.mtu_discovery.probes);
-	fprintf(stderr, "Probes total length took for calculating PMTU discovery : %d\n", node_pmtu.mtu_discovery.probes_total_len);
-	fprintf(stderr, "Time took for calculating PMTU discovery : %lu\n", node_pmtu.mtu_discovery.time);
-	fprintf(stderr, "Total MTU ping probes : %d\n", node_pmtu.mtu_ping.probes);
-	fprintf(stderr, "Total MTU ping probes length : %d\n", node_pmtu.mtu_ping.probes_total_len);
+static void print_mtu_calc(pmtu_attr_t *node_pmtu) {
```

Comment:
If a function is not modifying its arguments, you should make them const:
`static void print_mtu_calc(const pmtu_attr_t *node_pmtu)`